template/scripts/buildWebpack.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
```

Comment:
you could use one of existing implementations without inventing own conventions (like passing ENV after command)